draft-duke-quic-load-balancers.md

```diff
@@ -505,6 +518,208 @@ reserved for config rotation {{config-rotation}}.
 The server then encrypts the second through seventeenth octets using the 128-bit
 AES-ECB cipher.
 
+# Retry Service {#retry-offload}
+
+When a server is under load, QUIC allows it to defer storage of connection state
+until the client proves it can receive packets at its advertised IP address.
+Through the use of a Retry packet, a token in subsequent client Initial
+packets, and the original_connection_id transport parameter, servers verify
+address ownership and clients verify that there is no "man in the middle"
+generating Retry packets.
+
+There are two different mechanisms to allow offload of DoS mitigation to a
+trusted network service. One requires no shared state; the server need only be
+configured to trust a retry service, though this imposes other operational
+constraints. The other requires shared key, but has no such constraints.
+
+## Common Requirements
```

Comment:
I think it should be clear that all non-Initial packets are untouched by both mechanisms.